resources/logging/templates/loki/servicerole.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.istio.io/v1alpha1
+kind: ServiceRole
+metadata:
+  name: {{ template "loki.fullname" . }}
+  labels:
+    app: {{ template "loki.name" . }}
+    chart: {{ template "loki.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  rules:
+  - services: ["logging.kyma-system.svc.cluster.local"]
+    methods: ["GET", "POST", "OPTIONS"]
+    paths: ["/api/*"]
+  - services: ["logging.kyma-system.svc.cluster.local"]
```

Comment:
service name should be updated to logging-loki

Reply:
using proper template now